src/sentry/sentry_metrics/indexer/id_generator.py

```diff
@@ -19,32 +20,32 @@ def reverse_bits(number: int, bit_size: int) -> int:
 
 # we will have room b/n version and time since for a while
 # so let's reverse the version bits to grow to the right
-# instead of left should we need more than 4bits for version
-#
-#  version             time since
-# (0100    | 00000001001000101000000111100011)
```

Comment:
why delete this? it's still relevant just needs to be tweaked

```python
# 0bit  version             time since
# (0   | 010    | 00000001001000101000000111100011)
# (0   | 1000   |  0000001001000101000000111100011)
```

Reply:
I was a bit confused by it. Why is the second one 1000? Isn't it supposed to be 3 bits?

Reply:
I thought the point of reversing the bits for the version was to allow for increasing to more bits if needed, this was just to show how if we allow more bits for the version, we lose bits on the time since